hknweb/init_permissions.py

```diff
@@ -0,0 +1,44 @@
+from hknweb.candidate.models import Announcement, OffChallenge, BitByteActivity
+from hknweb.events.models import Event, Rsvp
+from django.contrib.auth.models import User, Group, Permission
+from django.contrib.contenttypes.models import ContentType
+
+candidate = Group.objects.get_or_create(name='candidate')[0]
+officer = Group.objects.get_or_create(name='officer')[0]
+
+#CANDIDATE AND OFFICER PERMISSIONS
+announcement = ('view_announcement', Announcement)
+add_offchallenge = ('add_offchallenge', OffChallenge)
+view_offchallenge =	('view_offchallenge', OffChallenge)
+change_offchallenge = ('change_offchallenge', OffChallenge)
+add_bitbyteactivity = ('add_bitbyteactivity', BitByteActivity)
+view_event = ('view_event', Event)
+add_rsvp = ('add_rsvp',Rsvp)
+remove_rsvp = ('delete_rsvp',Rsvp)
+view_user = ('view_user', User)
+
+cand_permission_names = [announcement, add_offchallenge, view_offchallenge, \
+	change_offchallenge, add_bitbyteactivity, add_rsvp, remove_rsvp, view_user]
+
+# OFFICER ONLY PERMISSIONS
+add_event = ('add_event', Event)
+change_event = ('change_event', Event)
+
+off_permission_names = [add_event, change_event]
+
+# SETTING PERMISSIONS TO GROUPS
+cand_permissions = []
+for perm in cand_permission_names:
+	ct =  ContentType.objects.get_for_model(perm[1])
+	permission = Permission.objects.get(codename=perm[0], content_type=ct)
+	cand_permissions.append(permission)
+
+off_permissions = cand_permissions
+for perm in off_permission_names:
+	ct =  ContentType.objects.get_for_model(perm[1])
+	permission = Permission.objects.get(codename=perm[0], content_type=ct)
+	off_permissions.append(permission)
```

Comment:
Ok real talk why are you using tabs?

... I know this is pedantic and will cause the usual flame wars but the rest of the codebase already uses 4 spaces instead of a tab. (I initially picked it based off of [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation), and muahaha now I get to argue for consistency.)

```suggestion
	off_permissions.append(permission)
```